dev/cd/after-branch-push

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
nit: use `.sh` suffix for the file name of these bash scripts

Reply:
Seems reasonable if the intent is to keep the naming of this file generic and potentially language agnostic. An alternative is that we could always assume it is a shell entrypoint, and that shell entrypoint may end up invoking other scripts/binaries (just food for thought)

My reasoning for using the `.sh` extension is mainly for convention (so it's obvious what type of file it is from the file name)